text/0047-introduce-profile-context.md

```diff
@@ -0,0 +1,110 @@
+- Start Date: 2022-12-20
+- RFC Type: feature
+- RFC PR: https://github.com/getsentry/rfcs/pull/47
+- RFC Status: draft
+
+# Summary
+
+We want to introduce the concept of a profiling context, much like a tracing context, starting with
+transactions so we can easily relate a group of transactions back to the respective profile.
+
+# Motivation
+
+We already have a way to relate a group of profiles back to the respective transaction. And up to
+this point, we have been using this relationship to map single transaction at a time to the
+respective profile. This is now insufficient as we would like to be able to relate a group of
```

Comment:
Given the thread above about the 1:1 relationship between transactions and profiles, I'm confused by the mention of "a group of transactions" here. Is the relationship 1:n in theory but 1:1 in practice?

Reply:
Ahh sorry, I'll update the wording but the intention here is that we want to be map a set of N transaction ids to a set of N profile ids